### ISSUE TITLE
Add extension method simplifying the creation of test logger

### ISSUE THREAD

**Issue by feO2x** [trbenning/serilog-sinks-xunit#7]
First of all, thanks for this sink, I use it in my daily programming life.

I would like to add an extension method that simplifies the creation of `ILogger` instances that log to `ITestOutputHelper`. Usually, in my test projects, it looks like this:

```csharp
public static ILogger CreateTestLogger(this ITestOutputHelper output) =>
    new LoggerConfiguration().WriteTo.TestOutput(output).CreateLogger();
```

Of course, we could add optional parameters for e.g. `outputTemplate` and `LogEventLevel` to this method.

In the end, it could be used like this:

```csharp
public sealed class MyTestClass
{
    private readonly TheClassIWantToTest _testTarget;

    public MyTestClass(ITestOutputHelper output)
    {
        _testTarget = new TheClassIWantToTest(output.CreateTestLogger());
    }

    // Facts and Theories omitted for the sake of brevity
}
```

IMO, this would simplify the calling code tremendously. What's your opinion?

**Comment by trbenning** [comment]
I like it. Do you want to submit a pull request?

**Comment by feO2x** [comment]
Yes, I will implement this in the upcoming days if that's OK for you.

**Comment by feO2x** [comment]
Is it OK if I add these methods to the `TestOutputLoggerConfigurationExtensions` class?

**Comment by trbenning** [comment]
I think I'd prefer adding a new class called `TestOutputHelperExtensions`.

**Comment by trbenning** [comment]
Merged in 1.0.7
